src/components/BookList.js

```diff
@@ -1,29 +1,84 @@
-import React from 'react';
-import { useSelector, useDispatch } from 'react-redux';
+import axios from 'axios';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchBooks, removeBook } from '../redux/books/booksSlice';
 import CreateBook from './CreateBook';
-import { removeBook } from '../redux/books/booksSlice';
 
-// BookList.js
-
-// ...
+const API_ID = 'u3BFViY5zE36mpKmwqF4';
+const API_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
 
 function BookList() {
-  const books = useSelector((state) => state.books.books);
   const dispatch = useDispatch();
+  const library = useSelector((state) => state.books.books);
+  // const books = Object.values(library).reduce((acc, books) => acc.concat(books), []);
+  const books = Object.entries(library).map(([identifier, data]) => ({
+    identifier,
+    data,
+  }));
+
+  console.log(books);
+  const loading = useSelector((state) => state.books.loading);
+  const error = useSelector((state) => state.books.error);
+
+  // eslint-disable-next-line
+  const deleteResource = async (item_id) => {
+    try {
+      // eslint-disable-next-line
```

Comment:
- [ ] You are currently disabling eslint in this file, kindly fix the linters rather than disabling eslint 👍

Reply:
Due to the nature of how the API displays id, the property is item_id  from the Api , The linter requires me to use camelCase which is inevitable